selendroid-standalone/src/main/java/io/selendroid/standalone/server/handler/ProxyToDeviceHandler.java

```diff
@@ -83,6 +83,9 @@ public Response handleRequest(HttpRequest request, JSONObject payload) throws JS
     while (retries-- > 0) {
       try {
         response = proxyRequestToDevice(request, session, url, method);
+        if (response == null) { // Unknown command
+          return new SelendroidResponse(sessionId, StatusCode.UNKNOWN_COMMAND);
```

Comment:
Yes, we should, but most clients should handle this version fine as well if I'm correct. At least this version doesn't throw some serialized exception at you.

Returning a 404 will require more work [as the code that handles the `SelendroidResponse` down the stack](https://github.com/selendroid/selendroid/blob/97e063cd2a28db269b5049848d2e32fba5e6e4d9/selendroid-server-common/src/main/java/io/selendroid/server/common/BaseServlet.java#L123-L136) is not aware of status codes based on the `SelendroidResponse` at all yet.

If you're ok with it I propose we merge this PR (given that clients will handle it) and I'll create a new issue to fix returning the right HTTP status code as well.